package.json

```diff
@@ -218,7 +218,7 @@
     "typescript": "^5.7.0-dev.20240826",
     "util": "^0.12.4",
     "vscode-nls-dev": "^3.3.1",
-    "webpack": "^5.91.0",
+    "webpack": "^5.94.0",
```

Comment:
No update in our `yarn.lock`?

Reply:
Bumped in https://github.com/microsoft/vscode/pull/227456